config/sync/user.role.site_builder.yml

```diff
@@ -0,0 +1,117 @@
+uuid: 959f5492-24df-4212-9a9c-a0da88d464c4
+langcode: en
+status: true
+dependencies: {  }
+id: site_builder
+label: 'Site Builder'
+weight: -1
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access config_pages overview'
+  - 'access content overview'
+  - 'access contextual links'
+  - 'access devel information'
+  - 'access entity usage statistics'
+  - 'access environment indicator'
+  - 'access file_browser entity browser pages'
+  - 'access files overview'
+  - 'access image_browser entity browser pages'
+  - 'access media overview'
+  - 'access media_browser entity browser pages'
+  - 'access patterns page'
+  - 'access site in maintenance mode'
+  - 'access site reports'
+  - 'access taxonomy overview'
+  - 'access user profiles'
+  - 'access video_browser entity browser pages'
+  - 'admin classes'
+  - 'admin display suite'
+  - 'admin fields'
+  - 'administer block_content display'
+  - 'administer block_content fields'
+  - 'administer block_content form display'
+  - 'administer blocks'
+  - 'administer content types'
+  - 'administer crop'
+  - 'administer crop types'
+  - 'administer display modes'
+  - 'administer entity browsers'
+  - 'administer footer menu items'
+  - 'administer image styles'
+  - 'administer main menu items'
+  - 'administer menu'
```

Comment:
Do we really wanna give this permission? they could modify the admin menu with this permission. With the menu admin per menu module, the permissions restrict them to only main nav and footer nav

Reply:
This role is generally targeting our producers or someone at a similar level. I think we should be ok with granting this permission for this role. At this first pass I'd rather avoid adding another contrib module.

Reply:
we've already added menu admin per menu module. so we could just rely on that

Reply:
The module is already there. Ignore previous comment.

Reply:
Ignoring is what i do best